pkg/kustomize/kustomize.go

```diff
@@ -97,11 +96,15 @@ func execAdd(fsys filesys.FileSystem, path string, AllTypesFiles map[fileType][]
 	// Resource: kustomize resource add resource f
 	for resType, files := range AllTypesFiles {
 		kustomizeCmd := resType.GetCommand()
+		if resType == Patch {
+			// Removes patches already present in kustomization.yaml
+			files, err = checkPresence(files, fsys)
```

Comment:
It should be better to rename this function, to correctly describe what it is done. This function not only check presence, but also remove some file. So, it is more a filter if present or something similar. What do you think about?

Reply:
you are right, I proposed `filterPatchinKust`

Reply:
If I'm not too late maybe `filterPatchFiles` can be also a valid alternative

Reply:
done